agent/src/test/java/com/microsoft/applicationinsights/agent/internal/agent/EnterExitClassVisitorTest.java

```diff
@@ -21,10 +21,19 @@
 
 package com.microsoft.applicationinsights.agent.internal.agent;
 
+
+import com.microsoft.applicationinsights.agent.internal.coresync.InstrumentedClassType;
+import org.junit.*;
+import org.mockito.Mockito;
+
+
 public final class EnterExitClassVisitorTest {
+
+    // TODO fix these tests
+//    @Ignore
```

Comment:
I made an attempt to fix this test, but it was a time sink. Instead, I added a TODO comment for later.